### DISCUSSION TITLE
Autres activités sportives / Daily routine

### DISCUSSION THREAD

**Discussion by The-GeekFamily** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#40] [Ideas]
Bonjour, 
est-il envisageable dans FiveM de pouvoir jouer au tennis comme dans GTA/GTAOnline ? Cela pourrait être intégré à la daily routine et permettrait d'augmenter force et endurance en une seule activité effectuée à plusieurs.
Je ne fais que peu de dev (essentiellement du scripting à mes heures) je ne me rends donc pas compte de la charge que cela peut représenter, mais je veux bien m'y plonger pour étudier la chose si besoin.
Merci.

**Comment by NariieL** [comment]
Hello,

Je t'invite à explorer ce qui peut déjà exister sur https://forum.cfx.re/c/development/releases/7

De ce que j'en sais, le Tennis est pas très agréable à réaliser sur FiveM, en plus d'être bourré de desync. Mais qui sait ?

Ce n'est absolument pas prévu dans nos plans, nous n'allons donc pas perdre de temps à nous renseigner sur cela de notre côté.

L'ajout de divers mini-jeux est totalement la bienvenue, et on serait ravit de review vos créations !
